addons/binding/org.openhab.binding.regoheatpump/README.md

```diff
@@ -0,0 +1,221 @@
+| IVT 490                           | Rego 400 Serial    |
+| IVT Premiumline X, Optima/290-AW  | Rego 800, Can bus  |
+| IVT Greenline HE/HC/HA+Prem HQ/EQ | Rego 1000, Can bus |
+| NIBE xx45                         | EB100, RS-485      |
+| NIBE Fighter series               | Styr 2002, RS-485  |
+| Thermia Diplomat series           | 901510, i2c        |
+
+Above list is informational, please consult with the Husdata interface provider for further details.
+
+### Thing configuration
+
+Two connection types are supported:
+* TCP/IP and
+* serial (RS232).
+
+#### TCP/IP connection
+
+A transparent bridge between the Husdata interface and network (i.e. wifi) is used.
+
+There are many existing project providing such functionality, i.e. [ser2net](http://ser2net.sourceforge.net/).
+
+Configuration of the TCP/IP thing:
+
+ - address: the hostname/IP address of the transparent bridge on the local network - mandatory,
+ - tcpPort: the port number to use to connect to the transparent bridge - optional, defaults to 9265.
+
+Example thing definition:
+ ```
+ regoheatpump:ipHusdata:ivtIP [ address="192.168.2.50", tcpPort="9265" ]
+ ```
+
+#### Serial connection
+
+One can connect the Husdata interface directly to a computer running the OH.
+
+Parameters:
+
+ - portName: the name of the serial port on your computer - mandatory.
+
+Example thing definition:
+
+```
+regoheatpump:serialHusdata:ivtSerial [ portName="COM3" ]
+```
+
+
+### Channels
+
+Below is the list of supported channels, all values are read only:
+
+| H1 ID | Name                  | Channel Type ID                  | Item Type        |
+|-------|-----------------------|----------------------------------|------------------|
+| 001   | Radiator Return       | sensorValues#radiatorReturn      | Temperature      |
+| 002   | Radiator Forward      | sensorValues#radiatorForward     | Temperature      |
+| 003   | Heat carrier Return   | sensorValues#heatFluidIn         | Temperature      |
+| 004   | Heat carrier Forward  | sensorValues#heatFluidOut        | Temperature      |
+| 005   | Brine In / Evaporator | sensorValues#coldFluidIn         | Temperature      |
+| 006   | Brine Out / Condenser | sensorValues#coldFluidOut        | Temperature      |
+| 007   | Outdoor               | sensorValues#outdoor             | Temperature      |
+| 008   | Indoor                | sensorValues#indoor              | Temperature      |
+| 009   | Hot water 1 / Top     | sensorValues#hotWater            | Temperature      |
+| 00A   | Hot water 2 / Mid     | sensorValues#externalHotWater    | Temperature      |
+| 00B   | Hot gas / Compressor  | sensorValues#compressor          | Temperature      |
+| 00E   | Air intake            | sensorValues#airIntake           | Temperature      |
+| 011   | Pool                  | sensorValues#pool                | Temperature      |
+| 104   | Add heat status       | controlData#addHeatPower         | Number - kW or % |
```

Comment:
Each value has 4 bits that describe the value type - degrees, kwh, %, ... so the easiest would be to have two channels and update the one based on value type.

I also was thinking about auto discovery and came to same conclusion - it is much work, specially since i.e. I do not own a Husdata interface and added support solely based on the information from the doc I shared above so in order to test it properly, it could take a lot of time...